mgmtsystem_severity/tests/test_create_severity.py

```diff
@@ -0,0 +1,71 @@
+from openerp.tests.common import TransactionCase
+from psycopg2 import IntegrityError
+
+
+class TestCreateSeverity(TransactionCase):
+
+    """
+    Test management severity object.
+
+    Test the management severity object creation.
+    It checks that each fields are required and that
+    a valid value creates an entry.
+    """
+
+    def setUp(self):
+        super(TestCreateSeverity, self).setUp()
+
+        self.severities = self.registry('mgmtsystem.severity')
```

Comment:
Your registry attribute could be clearer if named `Severity` or `severity_model`
We could think `self.severities` is a list of browse records.